app/istqb-ctfl-v4-chapter2-sdlc-and-testing/istqb-ctfl-v4-chapter2-sdlc-and-testing.css

```
@@ -0,0 +1,555 @@
+}
+
+.istqb-ctfl-v4-chapter2-page section {
+    padding-top: 0;
+}
+
+.istqb-ctfl-v4-chapter2-page .hero {
+    min-height: 0;
+    display: block;
+    padding-top: 0;
+    margin-bottom: 48px;
+    padding: 40px;
```

Stylelint (17.14.0)

[error] Overridden property "padding-top" by shorthand "padding" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)